include/database/DBManager.php

```diff
@@ -1928,7 +1931,9 @@ public function pQuery($sql, $data = array())
     {
         $stmt = $this->prepareQuery($sql);
 
-        return $this->executePreparedQuery($stmt, $data);
+        $query = $this->createPreparedQuery($stmt, $data);
+
+        return $this->query($query);
```

Comment:
What does `$this->query()` do if it's given `false`. 🤔  Should maybe switch this up so there's a check for `$query` being false.